add-on/_locales/en/messages.json

```
@@ -36,59 +48,47 @@
     "description": "A label tooltip in Node status section of Browser Action pop-up (panel_statusSwarmPeersTitle)"
   },
   "panel_quickImport": {
-    "message": "Share files via IPFS",
+    "message": "Quick Import/Share...",
```

Elipsis may look bit better?
```suggestion
    "message": "Quick Import/Share…",
```

---

🙏